nodejs/eks/package.json

```diff
@@ -14,7 +14,7 @@
     "dependencies": {
         "@pulumi/aws": "5.10.0",
         "@pulumi/kubernetes": "3.20.2",
-        "@pulumi/pulumi": "^3.0.0",
+        "@pulumi/pulumi": "^3.47.0",
```

Comment:
Is 3.47.0 the minimum version with the fix or just the latest version?

Reply:
It's the minimum version with the fix, and currently the latest version. Why?

Reply:
Cool - yup that's good - just checking it's not overly-restrictive